kotlin/workflow-ui/core-android/src/main/java/com/squareup/workflow/ui/WorkflowLayout.kt

```diff
@@ -41,13 +40,17 @@ import kotlinx.coroutines.flow.onEach
 class WorkflowLayout(
   context: Context,
   attributeSet: AttributeSet? = null
-) : FrameLayout(context, attributeSet) {
+) : ViewGroup(context, attributeSet) {
   private val showing: WorkflowViewStub = WorkflowViewStub(context).also {
-    addView(it, ViewGroup.LayoutParams(MATCH_PARENT, MATCH_PARENT))
+    addView(it, LayoutParams(MATCH_PARENT, MATCH_PARENT))
   }
 
   private var restoredChildState: SparseArray<Parcelable>? = null
 
+  private val child: View
```

Comment:
Is it worth it to make this an internal extension val?

Reply:
If this is changed to my suggestion, it couldn't be. And if it's not changed, then I still think probably not because keeping these classes self-contained makes them easily-reproducable samples for third party containers. If we trigger the rule of three and are worried about DRY, then I think a better solution than just factoring out this one property is factoring out a base class that also implements the two `ViewGroup` methods as well.